libs/application/template-api-modules/src/lib/modules/templates/children-residence-change/children-residence-change.service.ts

```diff
@@ -153,5 +155,12 @@ export class ChildrenResidenceChangeService {
         (application as unknown) as Application,
       )
     }
+
+    if (counterParty.phoneNumber) {
+      await this.smsService.sendSms(
+        counterParty.phoneNumber,
+        'Borist hefur umsókn um breytt lögheimili barns.',
```

Comment:
Would it be possible to send a link to the application? Also the copy here should probably be revisited.

Reply:
Yes noted it in the description. I decided to have this rather bad than good so that we would defentely revisit it 😅 And yes we could send a link :)